stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -1018,12 +1033,17 @@
 ;; This method extends the `tx-sender`'s current lockup for an additional `extend-count`
 ;;    and associates `pox-addr` with the rewards
 (define-public (stack-extend (extend-count uint)
-                             (pox-addr { version: (buff 1), hashbytes: (buff 32) }))
+                             (pox-addr { version: (buff 1), hashbytes: (buff 32) })
+                             (updated-signing-key (optional (buff 33))))
    (let ((stacker-info (stx-account tx-sender))
          ;; to extend, there must already be an etry in the stacking-state
          (stacker-state (unwrap! (get-stacker-info tx-sender) (err ERR_STACK_EXTEND_NOT_LOCKED)))
          (amount-ustx (get locked stacker-info))
          (unlock-height (get unlock-height stacker-info))
+         (current-signing-key (unwrap! (get signing-key stacker-state) (err ERR_STACK_EXTEND_NOT_SIGNED)))
```

Comment:
This error case shouldn't be reachable because `stack-stx` requires a signing key.  Maybe `unwrap-panic` would be better here?